services/actions/src/rpc/genSchemasDocs.js

```diff
@@ -53,53 +54,88 @@ const setMarkdownDocMutation = `
   }
 `;
 
+const getMemberFieldsMapper = source => param => {
+  const entry = Object.entries(param)[0];
+
+  if (!entry) {
+    return null;
+  }
+
+  const [key, title] = entry;
+
+  if (!source[key]) {
+    return null;
+  }
+
+  if (Array.isArray(source[key])) {
+    return `${text(title, { bold: true, indent: 2, postNewLine: false })}: \`${source[key].join(', ')}\``;
+  }
+
+  return `${text(title, { bold: true, indent: 2, postNewLine: false })}: \`${source[key]}\``;
+};
+
+const generateMemberDoc = member => {
+  const { meta } = member;
+
+  let result = header(member.shortTitle, { size: 5, bold: true, indent: 8 });
+  result += text(member.description || 'No description provided', { indent: 8 });
+  result += text('Parameters:', { bold: true, indent: 8 });
+
+  const parameters = [
```

Comment:
we need to show the column name for dimensions

Reply:
there's `sql` field in the params. Is that what you mean?

Reply:
yes

Reply:
still not showing `sql` data